editor/src/components/navigator/navigator-item/navigator-item.tsx

```diff
@@ -187,45 +256,36 @@ export const NavigatorItem: React.FunctionComponent<NavigatorItemInnerProps> = b
     } = props
 
     const domElementRef = useScrollToThisIfSelected(selected)
+    const isFocusedComponent = useEditorState((store) => {
```

Comment:
I think it'd be worth making a function (in this file) that calculates `isFocusedComponent`, because we are doing similar but not the same logic in two places and I suspect that we might need to update this (because the dom-walker work is not 100% finished yet)

Reply:
:+1: